node/chainSimulator/components/statusComponents.go

```diff
@@ -44,9 +46,16 @@ func CreateStatusComponents(shardID uint32, appStatusHandler core.AppStatusHandl
 		statusPollingIntervalSec: statusPollingIntervalSec,
 	}
 
-	// TODO add drivers to index data
-	instance.outportHandler, err = outport.NewOutport(100*time.Millisecond, outportCfg.OutportConfig{
-		ShardID: shardID,
+	hostDriverArgs, err := makeHostDriversArgs(external)
+	if err != nil {
+		return nil, err
+	}
+	instance.outportHandler, err = factory.CreateOutport(&factory.OutportFactoryArgs{
+		IsImportDB:               false,
+		ShardID:                  shardID,
+		RetrialInterval:          100 * time.Millisecond,
+		HostDriversArgs:          hostDriverArgs,
+		EventNotifierFactoryArgs: &factory.EventNotifierFactoryArgs{},
```

Comment:
maybe fill the args required for notifier?

Reply:
In this case no, because we want everyone to use the host driver for the notifier and indexer